### DISCUSSION TITLE
Golden Soroban Quest Entries

### DISCUSSION THREAD

**Discussion by JuanConner** [stellar/sorobanathon#167] [General]
Hey everyone, Juan Conner here. Just finished up my Golden Soroban Quest to share with ya'll. It's been fun, and I hope we can do a part deux. My group had a blast learning NFTs through this opportunity.
Here are my submissions:

Prologue- The Quest Begins-
 https://litemint.com/items/GDB3VRV76WRQS7444XVMNNCOQWFRLR2X3YP3TKXMGNGGTMCL6I7E33QF/QuestBegins

Chapter 1-  Hello World-

https://litemint.com/items/GCYEYITLLXSN3F6QA25DVMPBS4HNRCMAACB2JOQU5BSALNFCFPSO6VSG/HelloWorld

Chapter 2- Hot and Slithery-
 https://litemint.com/items/GABAHVQTIENJA55W7TI5UWT5LUPOTTO5HFQU7AVY6JUSI3M36HQAI5ZY/HotSlithery

Chapter 3- The Fiery Barrier-
 https://litemint.com/items/GDFDDZ3Q547MDK2IQ5T2ASRI3PYPYUDJN7QPFWYHKNUBTUG5K4ZHEUEO/FieryBarrier

Chapter 4- Old Man Gatekeeper-
 https://litemint.com/items/GCIKKGMKAWPRURABZQCBERHPGCTK4UKVD5DIEBCSRF2ICDBX3GFEKUWG/Gatekeeper

Chapter 5- The Golden Soroban Awaits-
 https://litemint.com/items/GCEWOJPBG3OU75GPKPFZZV3ZR3AXLXZS4VPEBOOCGXHOLF3EXGGG35IC/SorobanAwait

Chapter 6- A Lucky Pathway-
 https://litemint.com/items/GBRUUGFC5LA4M3N4SZQWOZLOEX7WI3JY4ZVFO34ELPO6MATTTSAHSJKW/LuckyPathway

I look forward to seeing all your submissions! This was really fun. Good luck to everyone!

**Comment by stellarjohn** [comment]
Hey, is there any way you can drop your image files in here? Litemint isn't loading these mints for some reason.

**Reply by JuanConner** [reply]
There is a way and I'll do just that. Thank you for reaching out. I'm sending my form currently too. I'll be emailing a sworn affidavit for my identity as well as my driver's license, as my identity is a bit of a mess.

I had to get Miriam to confirm who I am recently. So if there's any discrepancies I hope that'll smooth it over.

Litemint was working just fine last night, but for some reason it does not want to pull any of my work up today. Not sure what's going on over there. But thanks for your patience. Everyone I've talked to had a blast working on this project. Kudos for putting it on.

**Comment by JuanConner** [comment]
QuestBegins


![2a9a71b2-81e2-444f-b80c-f02a9f2561ff](https://user-images.githubusercontent.com/120694602/208190498-14e14832-1451-42a1-9ad0-2864ec68506b.jpg)

**Comment by JuanConner** [comment]
HelloWorld

![ZomboMeme 15122022001916](https://user-images.githubusercontent.com/120694602/208190609-00b5a0a0-184d-46d1-9848-7da14e6e1576.jpg)

**Comment by JuanConner** [comment]
HotSlithery

![ZomboMeme 15122022145953](https://user-images.githubusercontent.com/120694602/208190875-7aa468ee-f369-406e-9bac-2590c600ab01.jpg)

**Comment by JuanConner** [comment]
FieryBarrier 

![ZomboMeme 15122022150137](https://user-images.githubusercontent.com/120694602/208191111-3e18b64a-5851-4119-a186-0273af1e5712.jpg)

**Comment by JuanConner** [comment]
Gatekeeper 

![ZomboMeme 15122022163918](https://user-images.githubusercontent.com/120694602/208191244-7b4333db-b673-4850-ae08-45e9de94fc72.jpg)

**Comment by JuanConner** [comment]
SorobanAwait

![cd4751b9-1852-46a0-88bf-b78dc82b5df5](https://user-images.githubusercontent.com/120694602/208191438-9e71abe4-f5de-4eb8-a7af-465faa80c877.jpg)

**Comment by JuanConner** [comment]
LuckyPathway 

![ZomboMeme 15122022173330](https://user-images.githubusercontent.com/120694602/208191575-5a5cee77-95d8-48b9-8739-4531385c0988.jpg)

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by JuanConner** [reply]
Ahhhhh did the last one get lost? I filled it out and submitted. I can do it again if that's the case.

**Reply by JuanConner** [reply]
> Congratulations! Your contribution to has been accepted! 🎉 Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
> 
> Please let us know if you have any questions.
> 
> Cheers! John

Hey John, Sorry I've been busy with the holidays. Totally didn't finish the entire form last time I submitted. I followed through and resubmitted this time around, everything looks like it processed at least. Thanks for your undying patience good sir. Let me know if i need to do anything else. Have a Merry Christmas!
